src/app/public/modules/page-summary/page-summary.component.ts

```diff
@@ -25,6 +28,9 @@ export class SkyPageSummaryComponent implements OnDestroy, AfterViewInit {
         this.adapter.updateKeyInfoLocation(this.elRef, args === SkyMediaBreakpoints.xs);
       }
     );
+    setTimeout(() => {
+      this.hasKeyInfo = this.keyInfoContainerEl.nativeElement.childNodes.length > 0;
```

Comment:
Also, might be worth putting this check in `ngAfterContentInit` since ng-content isn't registered until then.